-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-40914/34396

ERROR: Build errors found during clang-tidy run.
```
TrackingTools/DetLayers/src/ForwardDetLayer.cc:102:3: error: invalid preprocessing directive [clang-diagnostic-error]
# Disable bitwise-instead-of-logical warning, see discussion in
  ^
TrackingTools/DetLayers/src/ForwardDetLayer.cc:103:3: error: invalid preprocessing directive [clang-diagnostic-error]
# https://github.com/cms-sw/cmssw/issues/39105
  ^
Suppressed 510 warnings (509 in non-user code, 1 with check filters).
--
RecoPixelVertexing/PixelTriplets/plugins/PixelTripletLargeTipGenerator.cc:76:3: error: invalid preprocessing directive [clang-diagnostic-error]
# Disable bitwise-instead-of-logical warning, see discussion in
  ^
RecoPixelVertexing/PixelTriplets/plugins/PixelTripletLargeTipGenerator.cc:77:3: error: invalid preprocessing directive [clang-diagnostic-error]
# https://github.com/cms-sw/cmssw/issues/39105
  ^
TrackingTools/DetLayers/interface/rangesIntersect.h:13:3: error: invalid preprocessing directive [clang-diagnostic-error]
# Disable bitwise-instead-of-logical warning, see discussion in
  ^
TrackingTools/DetLayers/interface/rangesIntersect.h:14:3: error: invalid preprocessing directive [clang-diagnostic-error]
# https://github.com/cms-sw/cmssw/issues/39105
  ^
Suppressed 875 warnings (874 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/RecoPixelVertexing/PixelTriplets/plugins/PixelTripletLargeTipGenerator.cc
+++ b/RecoPixelVertexing/PixelTriplets/plugins/PixelTripletLargeTipGenerator.cc
@@ -73,8 +73,8 @@ void PixelTripletLargeTipGenerator::fillDescriptions(edm::ParameterSetDescriptio
   desc.add<double>("phiPreFiltering", 0.3);
 }
 
-# Disable bitwise-instead-of-logical warning, see discussion in
-# https://github.com/cms-sw/cmssw/issues/39105
+// Disable bitwise-instead-of-logical warning, see discussion in
+// https://github.com/cms-sw/cmssw/issues/39105
 
 #if defined(__clang__) && defined(__has_warning)
 #  if __has_warning("-Wbitwise-instead-of-logical")

--- a/RecoPixelVertexing/PixelTriplets/plugins/PixelTripletLargeTipGenerator.cc
+++ b/RecoPixelVertexing/PixelTriplets/plugins/PixelTripletLargeTipGenerator.cc
@@ -77,10 +77,10 @@ void PixelTripletLargeTipGenerator::fillDescriptions(edm::ParameterSetDescriptio
 // https://github.com/cms-sw/cmssw/issues/39105
 
 #if defined(__clang__) && defined(__has_warning)
-#  if __has_warning("-Wbitwise-instead-of-logical")
-#    pragma clang diagnostic push
-#    pragma clang diagnostic ignored "-Wbitwise-instead-of-logical"
-#  endif
+#if __has_warning("-Wbitwise-instead-of-logical")
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wbitwise-instead-of-logical"
+#endif
 #endif
 
 namespace {
@@ -96,9 +96,9 @@ namespace {
 }  // namespace
 
 #if defined(__clang__) && defined(__has_warning)
-#  if __has_warning("-Wbitwise-instead-of-logical")
-#    pragma clang diagnostic pop
-#  endif
+#if __has_warning("-Wbitwise-instead-of-logical")
+#pragma clang diagnostic pop
+#endif
 #endif
 
 void PixelTripletLargeTipGenerator::hitTriplets(const TrackingRegion& region,

--- a/TrackingTools/DetLayers/interface/rangesIntersect.h
+++ b/TrackingTools/DetLayers/interface/rangesIntersect.h
@@ -14,10 +14,10 @@
 // https://github.com/cms-sw/cmssw/issues/39105
 
 #if defined(__clang__) && defined(__has_warning)
-#  if __has_warning("-Wbitwise-instead-of-logical")
-#    pragma clang diagnostic push
-#    pragma clang diagnostic ignored "-Wbitwise-instead-of-logical"
-#  endif
+#if __has_warning("-Wbitwise-instead-of-logical")
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wbitwise-instead-of-logical"
+#endif
 #endif
 
 template <typename Range>
@@ -35,9 +35,9 @@ inline bool rangesIntersect(const Range& a, const Range& b, bool (*less)(T, T)) 
 }
 
 #if defined(__clang__) && defined(__has_warning)
-#  if __has_warning("-Wbitwise-instead-of-logical")
-#    pragma clang diagnostic pop
-#  endif
+#if __has_warning("-Wbitwise-instead-of-logical")
+#pragma clang diagnostic pop
+#endif
 #endif
 
 #endif

--- a/TrackingTools/DetLayers/interface/rangesIntersect.h
+++ b/TrackingTools/DetLayers/interface/rangesIntersect.h
@@ -10,8 +10,8 @@
  *  the interval and r.second is the end of the interval.
  */
 
-# Disable bitwise-instead-of-logical warning, see discussion in
-# https://github.com/cms-sw/cmssw/issues/39105
+// Disable bitwise-instead-of-logical warning, see discussion in
+// https://github.com/cms-sw/cmssw/issues/39105
 
 #if defined(__clang__) && defined(__has_warning)
 #  if __has_warning("-Wbitwise-instead-of-logical")

--- a/TrackingTools/DetLayers/src/ForwardDetLayer.cc
+++ b/TrackingTools/DetLayers/src/ForwardDetLayer.cc
@@ -99,23 +99,23 @@ pair<bool, TrajectoryStateOnSurface> ForwardDetLayer::compatible(const Trajector
   // check r
   auto r2 = myState.localPosition().perp2();
 
-// Disable bitwise-instead-of-logical warning, see discussion in
-// https://github.com/cms-sw/cmssw/issues/39105
+  // Disable bitwise-instead-of-logical warning, see discussion in
+  // https://github.com/cms-sw/cmssw/issues/39105
 
 #if defined(__clang__) && defined(__has_warning)
-#  if __has_warning("-Wbitwise-instead-of-logical")
-#    pragma clang diagnostic push
-#    pragma clang diagnostic ignored "-Wbitwise-instead-of-logical"
-#  endif
+#if __has_warning("-Wbitwise-instead-of-logical")
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wbitwise-instead-of-logical"
+#endif
 #endif
 
   if ((r2 > rmin() * rmin()) & (r2 < rmax() * rmax()))
     return make_pair(true, myState);
 
 #if defined(__clang__) && defined(__has_warning)
-#  if __has_warning("-Wbitwise-instead-of-logical")
-#    pragma clang diagnostic pop
-#  endif
+#if __has_warning("-Wbitwise-instead-of-logical")
+#pragma clang diagnostic pop
+#endif
 #endif
 
   // take into account the thickness of the layer

--- a/TrackingTools/DetLayers/src/ForwardDetLayer.cc
+++ b/TrackingTools/DetLayers/src/ForwardDetLayer.cc
@@ -99,8 +99,8 @@ pair<bool, TrajectoryStateOnSurface> ForwardDetLayer::compatible(const Trajector
   // check r
   auto r2 = myState.localPosition().perp2();
 
-# Disable bitwise-instead-of-logical warning, see discussion in
-# https://github.com/cms-sw/cmssw/issues/39105
+// Disable bitwise-instead-of-logical warning, see discussion in
+// https://github.com/cms-sw/cmssw/issues/39105
 
 #if defined(__clang__) && defined(__has_warning)
 #  if __has_warning("-Wbitwise-instead-of-logical")

